docs/manuals/studio/config.md

```diff
@@ -1,8 +1,188 @@
 # Editing configuration files
 
-[TODO]
+Applications that rely on [Taipy Core](../core/index.md) need to define
+a [Configuration](../core/config/index.md) where the data flow is
+described.<br/>
+This configuration can be coded in Python, tuned using environment variables,
+or be expressed as a TOML file as described in the
+[TOML file configuration](../core/config/advanced-config.md#explicit-toml-file-configuration)
+section.
```

Comment:
Two remarks: 

1. Vincent and I recently made some updates to the page you are referring to. I just pushed the changes. You link is no longer up-to-date.
 
2. What you are saying is not really the truth. The section you mention is made to document how to override a python code configuration using TOML file, And not how to build a complete configuration in TOML.

I would not say that we can build a configuration using TOML. I would just say either we use Python code, or we use Studio and load the TOML result generated by Studio.

```suggestion
This configuration can be coded in Python, tuned using environment variables
(as described in the [advanced configuration](../core/config/advanced-config.md#Python-code-configuration) section).
```